File: Front_Angular_Pathfinder/src/app/core/models/ItemCuadernilloDTO.ts
import { ClaseDTO } from "./ClaseDTO";
import { ConquistadorDTO } from "./ConquistadorDTO";
import { CronogramaDTO } from "./CronogramaDTO";

export interface ItemCuadernilloDTO{
    ItcuId: number;
    ItcuDescripcion: string;
    ItcuClase: ClaseDTO;
    ItcuCronogramas: CronogramaDTO;
    ItcuConquistadores: ConquistadorDTO;
}

File: Back_Net8_Pathfinder2/Core/DTO/ItemCuadernilloDTO.cs
namespace Core.DTO;

public class ItemCuadernilloDTO
{
    public int ItcuId { get; set; }
    public string ItcuDescripcion { get; set; }
    public ClaseDTO ItcuClase { get; set; }
    public ICollection<CronogramaDTO> ItcuCronogramas { get; set; }
    public ICollection<ConquistadorDTO> ItcuConquistadores { get; set; }
}